packages/pwa/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -18,12 +18,22 @@ import {noop} from '../../../../utils/utils'
 import {API_ERROR_MESSAGE} from '../../../../constants'
 
 export const REMOVE_WISHLIST_ITEM_CONFIRMATION_DIALOG_CONFIG = {
```

Comment:
Thought: Should we move this to `constant.js` as we already have similar type of constant in there? (e.g REMOVE_CART_ITEM_CONFIRMATION_DIAGLOG_CONFIG)

Reply:
I think I'll just leave it there. Looks like it's meant to be used in the same file. So I'm not sure why it was being exported. But yeah, I'll just leave it as it is :)